### ISSUE TITLE
kcdshop crash when clicking on ''START APP"

### ISSUE THREAD

**Issue by onemen** [epicweb-dev/kcdshop#6]
![app_crushed](https://user-images.githubusercontent.com/3650909/225653699-365bebe8-1927-4967-8ebe-01e5af774387.png)

kcdshop crush when clicking on ''START APP" in `problem` or `solution` in windows.

**Comment by onemen** [comment]
Adding `shell: true` in process-manager.server.ts 'runAppDev' solved this issue
```diff
	const appProcess = spawn('npm', ['run', 'dev'], {
		cwd: app.fullPath,
		shell: true,
		env: {
			...process.env,
			// TODO: support specifying the env
			NODE_ENV: 'development',
			// TODO: support specifying the port
			PORT: String(portNumber),
			// let it pick a random port...
			REMIX_DEV_SERVER_WS_PORT: '',
		},
	})
```

Maybe you should use `shell: true` for all spawn process on windows?

**Comment by onemen** [comment]
fixed by eab911b

911 to the rescue
